app/src/main/java/org/mozilla/focus/menu/context/WebContextMenu.java

```diff
@@ -90,6 +92,12 @@ public void onCancel(DialogInterface dialog) {
         dialog.getWindow().setBackgroundDrawable(new ColorDrawable(Color.TRANSPARENT));
 
         final NavigationView menu = (NavigationView) view.findViewById(R.id.context_menu);
+        NavigationMenuView navigationMenuView = (NavigationMenuView) menu.getChildAt(0);
```

Comment:
Same here.

Reply:
This one does have to be done dynamically https://stackoverflow.com/questions/32673303/remove-scrollbar-from-android-support-design-navigation-drawer